bench/static_analysis/LuauPolyfillMap.lua

```diff
@@ -0,0 +1,961 @@
+--!nonstrict
```

Comment:
Is this file MIT licensed? We might need a separate MIT header for this.

Reply:
Yes it is, adding the same header as the python version

Reply:
@zeux , added the header, would love to know if that suffices